youtube-tracker/README.md

```diff
@@ -0,0 +1,51 @@
+# Track YouTube video views and comments
+
+_Tracks views, likes, and comments for provided YouTube URLs in a Google Sheet with optional email notifications._
+
+Last updated: August, 2019
+
+Track performance of public YouTube videos, including views, likes, and comments, in a Google Sheet. 
+Refresh data automatically via the YouTube Data API. Send automatic email notifications
+that summarize new YouTube video comment activity.
+
+![screenshot](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/youtube-tracker/screenshot.png)
+
+## Technology highlights
+- Use the [YouTube advanced service](https://developers.google.com/apps-script/advanced/youtube) to collect data on specific videos (such as number of views and comment data) via the YouTube Data API.
+- Use the [Gmail Service](https://developers.google.com/apps-script/reference/gmail/) to send notification emails when new video comments are detected. 
+- Create an [HTML template](https://developers.google.com/apps-script/guides/html/templates#calling_apps_script_functions_from_a_template) containing an Apps Script function to send emails containing HTML tables.
+
+## Try it
+
+### Spreadsheet setup
+1. Make of copy of the spreadsheet [here](https://docs.google.com/spreadsheets/d/12rQe1ndU_VmmHl0QIqUi-XxQ8lWovjh0xfOHTfxOHoo/copy). It already contains the Apps Script code from this repository.
+2. Change the name of the tab to the full email address where you’d like to receive email notifications. 
+3. Locate URLs of videos you would like to track and add them in column A below cell A1.
+
+### YouTube Advanced Service setup
+1. From the spreadsheet, open the script editor by selecting Tools > Script editor.
+2. In the script editor, select Resources > Advanced Google services.
```

Comment:
FYI, in markdown it's traditional to use `1.` for all bullets in a numbered list. The rendered will assign the correct number, and then you can reorder the list more easily later.

Reply:
Cool! And done